src/document_plot.py

```diff
@@ -0,0 +1,31 @@
+import os
+import pytest
+from jina import Document
+from jina.helper import random_identity
+
+from .utils.benchmark import benchmark_time
+from .pages import Pages
+
+
+random_identity(use_uuid1=True)
+
+
+@pytest.mark.parametrize("num_docs", [1, 5])
+def test_document_plot(num_docs, json_writer, ephemeral_tmpdir):
+    def _input_docs():
+        return (
+            (),
+            dict(docs=[Document(text="doc") for _ in range(num_docs)]),
+        )
+
+    def _plot(docs):
+        for d in docs:
+            d.plot(output=os.path.join(ephemeral_tmpdir, "doc.svg"))
```

Comment:
Are you sure, we should define `output`? this will put quite some traffic on mermaid.ink and we will only test their service inside ours. I'd rather remove this.

Reply:
Agree but if no otuput is given are we really testing .plot?

Reply:
This is not about testing `plot` but benchmarking our side of it. I'd say its fine